src/index.html

```diff
@@ -11,7 +11,171 @@
+        >
+        </label>
+
+        <label class="form-field">
+        How old are You?
+          <input
+        class="input"
+        type="number"
+        name="age"
+        min="1"
+        max="100"
+        value="12"
+        >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+        class="input"
+        type="date"
+        name="date"
+        >
+        </label>
+
+        <label class="form-field">
+      I accept the term of the agreement
+          <input
+    class="input"
+    type="checkbox"
+    name="agreement"
+    >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset field-border">
+        <legend>Registration</legend>
+        <label class="form-field">
+        E-mail:
+          <input
+      class="input"
+      type="email"
+      name="email"
+      placeholder="email@example.com"
+        >
+        </label>
+
+        <label class="form-field">
+        Password:
+          <input
+        class="input" type="password"
+        name="password" required
+        minlength="5" maxlength="10"
+        >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset field-border">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-field radio">
+        Do you love cats?
+
+          <input
+        class="input"
+        type="radio"
+        name="loveCats"> Yes
+
+          <input
+        class="input"
+        type="radio"
+        name="loveCats"> No
+
+        </label>
+
+        <label class="form-field">
+        What is your favorite color?
+          <input
+        class="input"
+        type="color"
+        name="color"
+          >
+        </label>
+
+        <label class="form-field">
+        What time do you go to bed?
+          <input
+        class="input"
+        type="time"
+        name="sleepTime"
+        step="2"
+          >
+        </label>
+
+        <label class="form-field">
+
+        What are your favorite brand of cars?
+          <select name="brand" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="volkswagen">Lada</option>
+          </select>
+        </label>
+
+        <label for="workRate">How do you rate our work?
+          <input
+          class="input" type="range"
+          id="workRate" name="workRate"
+              min="0" max="11"
+              value="0">
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset field-border">
+        <legend>Additional info:</legend>
+        <label class="form-field">
+        Comments:
+          <textarea
+        class="input" name="feedback"
+        cols="20" rows="2">
+          </textarea>
+        </label>
+
+        <label class="form-field">
+          Would you recommend us?
+          <select name="recommend" class="input">
+
+            <option value="Yes">yes</option>
```

Comment:
```suggestion
          <select name="recommend" class="input">
            <option value="Yes">yes</option>
```
не оставляй пустых строк между родительским и дочерним тегами